pandas/tests/resample/test_datetime_index.py

```diff
@@ -752,7 +752,8 @@ def test_resample_origin():
     resampled = ts.resample("5min", origin=offset_timestamp).mean()
     tm.assert_index_equal(resampled.index, exp_rng)
 
-    resampled = ts.resample("5min", origin="epoch", offset="2m").mean()
+    with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
why are you catching this? why does this have a warning?

Reply:
I thought we shouldnt change tests, hence the assert. Now I removed the assert and changed m to min.